harness/harness.go

```diff
@@ -464,15 +485,22 @@ func (t *H) report() {
 	}
 	dstr := fmtDuration(t.duration)
 	format := "--- %s: %s (%s)\n"
+
+	var status string
 	if t.Failed() {
```

Comment:
I think this is more readable if we split up the status setting (or make a helper) and the flushing.

Making it `if t.Failed() || t.suite.opts.Verbose { flushToParent }` by itself makes the reason for that much more obvious.

What you have now  is already an improvement over what it was before, but I think it can still be a little better.